_config.yml

```diff
@@ -164,11 +164,17 @@ redirect_from:
   json: false
 
 help_pages:
```

Comment:
There's a reference to this that I'm unsure if would be affected by this, but it seems like this helper is totally unused anyways? (Can't find any references to `translate_page` in code)

https://github.com/GSA-TTS/identity-site/blob/76d68b9f5ac8aba763ca263f1e003eff4c9d401c/_includes/help/translate_page.html#L4

Reply:
Removed the `translate_page` file.